docs/api.md

```diff
@@ -0,0 +1,392 @@
+# Ocean+ Habitats API
+
+## Licensing
+The Ocean+ Habitats API does not currently require any licensing or tokens to use.
+
+## Endpoints
+The Ocean+ Habitats API exposes statistics regarding protected area coverage, habitat coverage change and Species Red List statuses.
+
+This information can be requested either grouped by country or for a given region.
+
+### Countries
+Country-level data can be requested grouped by country or for a specific country.
```

Comment:
'grouped by country' is also a bit confusing. i'd just drop this sentence, line 14 is clear on what's available